recipes/zip/all/conanfile.py

```diff
@@ -0,0 +1,62 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class ZipConan(ConanFile):
+    name = "zip"
```

Comment:
urgh. I really don't like that this lib is named after a file format which it did not invent

Reply:
neither me

Reply:
We have our `author-repo` name which could be applied?

Reply:
Agreed. We applied the `authorname-projectname` several times in the past when we were thinking that the name is too generic

Reply:
Yes! `kuba-zip` please

Reply:
renamed